.github/ISSUE_TEMPLATE/4.docs-feedback.yml

```diff
@@ -0,0 +1,52 @@
+name: Docs feedback
+description: Improve documentation about MUI Core.
+labels: ['status: needs triage', 'docs']
+title: '[docs] '
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please provide a searchable summary of the issue in the title above ⬆️.
+
+        Thanks for contributing by creating an issue! ❤️
+  - type: checkboxes
+    attributes:
+      label: Duplicates
+      description: Please [search the history](https://github.com/mui/material-ui/issues) to see if an issue already exists for the same problem.
+      options:
+        - label: I have searched the existing issues
+          required: true
+
+  - type: input
+    id: page-url
+    attributes:
+      label: Related page
+      description: Which page of the documentation is concerned?
+      placeholder: https://mui.com/material-ui/react-badge/
+    validations:
+      required: true
+
+  - type: dropdown
+    attributes:
+      label: Kind of issue
+      description: What version of our software are you running?
+      options:
+        - Unclear explanations
+        - Missing information
+        - Broken demonstration
+        - Other
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Issue description
+      description: |
+        Describe as precisely, what went wrong during your experience with this documentation page and what could we do to improve it.
```

Comment:
```suggestion
        Let us know what went wrong when you were using this doc and what we could do to improve it.
```

Reply:
Thought that documentation was abbreviated "docs" instead of "doc"